week1-2/views/MyAccountView.js

```diff
@@ -0,0 +1,68 @@
+import { eventController } from "../EventController.js";
+import { storage } from "../Storage.js";
+import { qs } from "../util/domHelper.js";
+import View from "./View.js";
+
+const tag = "[myAccountView]";
+export default class MyAccountView extends View {
+  constructor() {
+    console.log(tag, "[myAccountView]");
+    super(qs(".all_my_amount"));
+    this.plusAmount = qs(".amount_detail_plus", this.element);
+    this.minusAmount = qs(".amount_detail_minus", this.element);
+
+    this.totalMyAccount(storage.HISTORY_LIST);
+    this.detailPlusAndMinus(storage.HISTORY_LIST);
+
+    //이벤트 발행(firstRender함수의 on메서드에서 이벤트 수신)
+    eventController.subscribe("updatedList", (updatedList) => {
+      console.log("이벤트발행", updatedList);
+      this.totalMyAccount(updatedList);
+      this.detailPlusAndMinus(updatedList);
+    });
+  }
+
+  calculateAccount(items) {
+    let totalIncome = 0;
+    let totalSpend = 0;
+
+    items.forEach((item) => {
+      if (item.SPEND_OR_INCOME === "income") {
+        totalIncome += item.PRICE;
+      } else if (item.SPEND_OR_INCOME === "spend") {
+        totalSpend += item.PRICE;
+      }
+    });
```

Comment:
3항 연산자로 구현한다면, 아래처럼 나타낼 수 있을 것 같네용 !
```suggestion
    items.forEach((item) => {
      item.SPEND_OR_INCOME === "income" ? totalIncome += item.PRICE : totalSpend += item.PRICE;
    });
```

Reply:
오 else if 도 삼항연산자 되는구나?? 나는 if ~ else만 되는건줄 알았어 ㅋㅋㅋㅋㅋㅋ 고마워 아름